contracts/forwarder/Forwarder.sol

```diff
@@ -142,24 +142,24 @@ contract Forwarder is IForwarder {
     function _getEncoded(
         ForwardRequest calldata req,
         bytes32 requestTypeHash,
-        bytes memory suffixData
+        bytes calldata suffixData
     )
     public
     pure
     returns (
         bytes memory
     ) {
+        // we use encodePacked since we append suffixData as-is, not as dynamic param.
+        // still, we must make sure all first 6 params are encoded as 256-wide params.
```

Comment:
Six? I counted 7, 8 if you count requestTypeHash...

Reply:
Also, please add something like "256 bytes wide params in order to be a valid RLP encoded data". Also, maybe use natspec to add this comment to the method itself, I guess...